podcasts/End of Year/Stories/ListeningTimeStory.swift

```diff
@@ -78,7 +78,7 @@ struct ListeningTimeStory: StoryView {
 /// Apply a perspective to the podcasts cover
 struct PodcastCoverPerspective: ViewModifier {
     private var transform: CGAffineTransform {
-        let values: [CGFloat] = [0.89, 0, 0.58, 1, 0, 0]
+        let values: [CGFloat] = [1, 0, 0.40, 1, 0, 0]
```

Comment:
I changed the perspective effect because these values seems to provide a better final result.